docs/guides/postinstall.md

```diff
@@ -83,6 +83,16 @@ The steps to be followed vary depending upon the initramfs module loading mechan
          ```
 
     3. Run `sudo mkinitcpio -P`.
+ 
+- On EndeavourOS, which uses `dracut`
```

Comment:
Better to use this line:

On systems with `dracut` (Commonly used on EndeavourOS and Fedora):